modules/Adding-State-to-HTTP/exercises/Remember-Me-App/README.md

```diff
@@ -7,7 +7,7 @@ cookies to remember the name of visitors to our app.
 
 1. Create an empty directory & repository
 0. Run `npm init -y`
-0. Run `npm install --save express cookie-parser`
+0. Run `npm install --save express body-parser`
```

Comment:
This exercise was written as a draft and never fully fleshed out. I've removed the link to it for now. It really should require you to read and write cookie strings. I'll update this and put the link back to it soon.